.env

```diff
@@ -0,0 +1,2 @@
+STATIC_URL="http://localhost:3000/"
+BASE_PORT1=3000
```

Comment:
lets update this to be moe explicit what it is see https://github.com/wildone/synology-docker/blob/main/.env